ckanext/stadtzhtheme/dcat/profiles.py

```diff
@@ -279,8 +286,6 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):
 
             #  Lists
             items = [
-                ('theme', DCAT.theme, None),
-                ('language', DCT.language, None),
```

Comment:
I realized that DCT.language was already set in line 135. Setting DCAT.theme was also a duplicate. We set language like so: g.add((dataset_ref, DCT.language, Literal(ckan_locale_default)))